config/routes.rb

```diff
@@ -3,5 +3,6 @@
 
   resource :home, controller: :home
   resources :events, only: [:index, :show]
+  get ':name.:ext', to: redirect('/assets/%{name}.%{ext}'), constraints: { name: /.+/, ext: /(jpg|png|gif)/}
```

Comment:
I'd probably avoid creating separate routes for images by having proper paths set-up in HTML.